pipeline/config.go

```diff
@@ -24,14 +24,20 @@ type Config []operator.Config
 // BuildOperators builds the operators from the list of configs into operators
 func (c Config) BuildOperators(bc operator.BuildContext) ([]operator.Operator, error) {
 	operators := make([]operator.Operator, 0, len(c))
-	for i, builder := range c {
-		nbc := getBuildContextWithDefaultOutput(c, i, bc)
-		op, err := builder.Build(nbc)
+	pluginMap := make(map[string]string)
```

Comment:
Please add a comment explaining the usage of this map

Reply:
Added comment to explain map.